docs/self-managed/components/orchestration-cluster/core-settings/concepts/elasticsearch-and-opensearch.md

```diff
@@ -0,0 +1,97 @@
+- **SSL/TLS**: SSL settings such as `self-signed` or disabling hostname verification should only be used if connection issues occur.
+- **JVM certificates**: Certificates may need to be imported into the JVM runtime.
+
+You can specify either `host` and `port` (deprecated) or `url` (recommended).
+
+## Common configuration options
+
+:::note
+`.{ES/OS}` in the values below reference either `.elasticsearch` or `.opensearch`.
+:::
+
+| Name                                             | Description                                  | Default value                                  |
+| ------------------------------------------------ | -------------------------------------------- | ---------------------------------------------- |
+| `camunda.data.{ES/OS}.cluster-name`              | Name of the Elasticsearch/OpenSearch cluster | elasticsearch / opensearch                     |
+| `camunda.data.{ES/OS}.url`                       | URL of the cluster REST API                  | [http://localhost:9200](http://localhost:9200) |
+| `camunda.data.{ES/OS}.username`                  | Username to access the cluster REST API      | -                                              |
+| `camunda.data.{ES/OS}.password`                  | Password to access the cluster REST API      | -                                              |
+| `camunda.data.{ES/OS}.security.certificate-path` | Path to SSL certificate                      | -                                              |
+| `camunda.data.{ES/OS}.security.self-signed`      | Certificate is self-signed                   | false                                          |
+| `camunda.data.{ES/OS}.security.verify-hostname`  | Validate hostname for SSL                    | false                                          |
+| awsEnabled                                       | Use AWS credentials (OpenSearch only)        | false                                          |
+| `camunda.data.{ES/OS}.index-prefix`              | Prefix for index names                       | operate / tasklist                             |
+| `camunda.data.{ES/OS}.number-of-shards`          | Number of shards for all indices             | 1                                              |
+| `camunda.data.{ES/OS}.number-of-replicas`        | Number of replicas for all indices           | 0                                              |
+
+:::note
+Shard and replica settings are applied on first startup or during version updates. After indices are created, they can be adjusted in the cluster template and applied to new indices.
+:::
+
+## Application configuration example
+
+```yaml
+camunda:
+  data:
+    secondary-storage:
+      elasticsearch:
+        cluster-name: elasticsearch
+        url: https://localhost:9200
+        security:
+          self-signed: true
+
+      opensearch:
+        cluster-name: opensearch
+        url: https://localhost:9200
+        security:
+          self-signed: true
```

Comment:
I find this example potentially confusing, given that we would only want our users to configure _either_ ES or OS, and not both. Maybe split it into two distinct examples?

Reply:
Happy to do that! ES💔OS